### ISSUE TITLE
Is there any plan to merge some of the feature/improvement branches back to dev?

### ISSUE THREAD

**Issue by ActivePHOENiX** [OrchardCMS/Orchard#7366]
I could see some work on some branches are very nice to have, some date back more than a year ago, I have merged the sqlite branch on my own branch, I wanna know is there any plan in near future to merge the useful branches to the master or dev branch officially or not, maybe merge is not a priority or totally defered/canceled because of the focus on the version 2 of this fantastic piece of software ...
the same is true for some pull requests like glimpse, that sebastian mentioned would be merged at a later time in a conversation with the contributor but never included or accepted. 
I would be glad to know for sure. thanks.

**Comment by sebastienros** [comment]
Please complain about and we'll take care of that. I actually thought the glimpse one was already merged on dev.

For the Sqlite one there might be some open issues, this is why I didn't merge.

**Comment by ActivePHOENiX** [comment]
I do not understand this sentence "Please complain about and we'll take care of that." was a serious feedback or just a sarcasm ... If it sounds like a complaint I do apologize , that was not my intention at all. I just asked for the sake of knowledge.

but all in all thank you for your time and clarification. I see that the issue has been added to the milestone.

Thanks for your consideration Sebastien.

**Comment by sebastienros** [comment]
It's not a complaint at all, it's serious, if you just remind us about the existence of these branches we might take the time to do that.
In the case of Sqlite, I was blocked by some limitations like the lack of table modification scripts. But if you can provide a PR that merges it in dev and it works fine then please do.
